sdk/servicebus/azure-servicebus/azure/servicebus/_common/receiver_mixins.py

```diff
@@ -38,6 +38,8 @@ def _create_attribute(self, **kwargs):
         )
         self._name = "SBReceiver-{}".format(uuid.uuid4())
         self._last_received_sequenced_number = None
+        self._message_iter = None
+        self._connection = kwargs.get("connection")
```

Comment:
self._prefetch = kwargs.get("prefetch")
?

Reply:
`prefetch` currently is being parsed and used through `Configuration` object -- `self._config.prefetch`

`self._prefetch` is not used.

Reply:
as per OOB discussion, moved "prefetch" and "idle_timeout" out of Configuration as they are receiver-only configs.